### PR TITLE
Return a default description when it is empty

### PR DESCRIPTION
**Pull request by sauloperez**
This fixes the following error spotted in Sidekiq web UI. See below

screenshot from 2018-09-27 13-13-35

### CODE REVIEW

File: spec/services/push_notifications/post_spec.rb [hunk 1]
```diff
@@ -13,10 +13,71 @@
...
 
   describe '#create!' do
-    it 'creates as many PushNotification resources as needed' do
-      expect {
-        creator.create!
-      }.to change{PushNotification.count}.by(1)
+    context 'integration' do
+      it 'creates as many PushNotification resources as needed' do
+        expect { creator.create! }.to change { PushNotification.count }.by(1)
+      end
```
Comment (sauloperez) [inline spec/services/push_notifications/post_spec.rb]: this test was very naive

**Comment by sauloperez** [comment]
[Investigar no enviament de push](https://trello.com/c/zyZFh25b/165-investigar-no-enviament-de-push)

**Comment by sauloperez** [comment]
Shall we invest some time in testing @sseerrggii ?

**Comment by sseerrggii** [comment]
I think we don't need more test here, le'ts merge it!

**Comment by sauloperez** [comment]
Talked offline. We might have tested this already but we'll double check now that I deployed it to staging.

**Comment by sseerrggii** [comment]
Tested! :heavy_check_mark:

**Pull request merged**